src/lib/src/common/workspace.spec.ts

```diff
@@ -348,56 +348,6 @@ describe('Workspace marker interface', () => {
     expect(actualMarker).toEqual({});
   });
 
-  it('removes stale markers on reload by default', () => {

```

Comment:
why were these expectations removed?

Reply:
… I managed to somehow mess up while rebasing the branch. Should be back in now.